backend/src/test/java/de/laura/project/controller/TriviaControllerTest.java

```diff
@@ -39,45 +45,104 @@ private String getUrl() {
     private TestRestTemplate testRestTemplate;
 
     @Autowired
-    private TriviaService triviaService;
+    private TempTriviaQuestionDB tempTriviaQuestionDB;
+
+    @BeforeEach
+    public void setup() {
+    tempTriviaQuestionDB.clear();
+    }
 
     @Test
-    @DisplayName("Get to /questions returns list of questions from Trivia API")
+    @DisplayName("Calls question set on the basis of the user's input")
 
-    public void returnQuestionSet() {
+    public void callQuestionSet() {
 
         //GIVEN
-        String url = "https://opentdb.com/api.php?amount=10&category=12&difficulty=medium&type=multiple";
-
-        ArrayList<String> answerForTriviaQuestionSet = new ArrayList<>();
-        answerForTriviaQuestionSet.add("correct answer");
-        answerForTriviaQuestionSet.add("answerOne");
-        answerForTriviaQuestionSet.add("answerTwo");
-        answerForTriviaQuestionSet.add("answerThree");
-
-        ArrayList<String> anotherAnswerForTriviaQuestionSet = new ArrayList<>();
-        answerForTriviaQuestionSet.add("another correct answer");
-        answerForTriviaQuestionSet.add("answerOne");
-        answerForTriviaQuestionSet.add("answerTwo");
-        answerForTriviaQuestionSet.add("answerThree");
+        String ApiUrl = "https://opentdb.com/api.php?amount=10&category=12&difficulty=medium&type=multiple";
```

Comment:
variable names should usually start with a lower case letter (camelCase) and not upper case (PascalCase)